jbpm-services/jbpm-services-api/src/build/revapi-config.json

```diff
@@ -42,6 +53,18 @@
         "old": "field org.jbpm.services.api.query.QueryResultMapper<T>.COLUMN_ORGANIZATIONAL_ENTITY",
         "new": "field org.jbpm.services.api.query.QueryResultMapper<T>.COLUMN_ORGANIZATIONAL_ENTITY",
         "justification": "Fix for some of dbs with query aliases"
+      },
+      {
+        "code": "java.field.enumConstantOrderChanged",
+        "old": "field org.jbpm.services.api.query.model.QueryDefinition.Target.CUSTOM",
+        "new": "field org.jbpm.services.api.query.model.QueryDefinition.Target.CUSTOM",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
```

Comment:
@mswiderski As to the relevance, unfortunately, iteration order of enums is part of the API. 
+1 on not committing without justifications, although I think they are better filled in by the committers of such breaking changes.